api-server/src/main/java/io/enmasse/api/server/RequestLogger.java

```diff
@@ -70,19 +61,21 @@ public void filter(ContainerRequestContext containerRequestContext, ContainerRes
     }
 
     private void updateMetrics(int status) {
-        if (status >= 400 && status < 500) {
-            num4xx++;
-        } else if (status >= 500) {
-            num5xx++;
+        if (status < 0 || status >= MAX_CODE) {
+            return;
         }
-        numTotal++;
+        statusCodeCounter[status]++;
```

Comment:
This isn't thread safe.

Reply:
Oops, meant to make that an array of atomics